compiler/infra/Assert.cpp

```diff
@@ -40,39 +40,37 @@
 
 void OMR_NORETURN TR::trap()
    {
-   static char * noDebug = feGetEnv("TR_NoDebuggerBreakPoint");
```

Comment:
Is the reason we are removing `static` because this code will only ever run once?

Reply:
It seemed like it was a weird detail that we didn't need.  Is using statics for environment variables standard practice in the compiler?

Reply:
> Is using statics for environment variables standard practice in the compiler?

It appears to be looking at other uses. I suspect we try to avoid the runtime overhead of extracting the variables every time. Sometimes the implementations may also print that a variable has been set, which would occur on every `feGetEnv` if it is not static. Of course we are about to abort here so the point is moot.

[1] https://github.com/eclipse/openj9/blob/da5a6a753bd1cd4dbb1384bd9ddccc942257c77e/runtime/compiler/runtime/Runtime.cpp#L1879-L1925

Reply:
I'll put it back to static, thanks for the help!

Reply:
Fixed now